backend/src/test/java/org/schmeckis/backend/RestControlleIntegrationTest.java

```diff
@@ -0,0 +1,340 @@
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import java.util.List;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@AutoConfigureMockMvc
+@SpringBootTest
+class RestControlleIntegrationTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private RezeptRepo rezeptRepo;
+
+    private final UUID idService = mock(UUID.class);
+
+
+    @Test
+    @DirtiesContext
+    void testGetAllRezepte() throws Exception {
+        //ARRANGE
+        List<Rezept> allRezepts = List.of(new Rezept("1", "Kartoffelsalat", "Kartoffelsalat mit Mayo", "Kartoffelsalat", "Kartoffelsalat",
+                List.of(new Kategorie("Salat", "Salat"))), new Rezept("2", "Nudelsalat", "Nudelsalat mit Mayo", "Nudelsalat", "Nudelsalat",
+                List.of(new Kategorie("Salat", "Salat"))));
+        rezeptRepo.save(allRezepts.get(0));
+        rezeptRepo.save(allRezepts.get(1));
+        String expected = """
+                [
+                    {
+                        "id": "1",
+                        "rezeptName": "Kartoffelsalat",
+                        "rezeptImageUrl": "Kartoffelsalat mit Mayo",
+                        "rezeptKurzbeschreibung": "Kartoffelsalat",
+                        "rezeptBeschreibung": "Kartoffelsalat",
+                        "kategorieList": [
+                            {
+                                "kategorieName": "Salat",
+                                "kategorieBeschreibung": "Salat"
+                            }
+                        ]
+                    },
+                    {
+                        "id": "2",
+                        "rezeptName": "Nudelsalat",
+                        "rezeptImageUrl": "Nudelsalat mit Mayo",
+                        "rezeptKurzbeschreibung": "Nudelsalat",
+                        "rezeptBeschreibung": "Nudelsalat",
+                        "kategorieList": [
+                            {
+                                "kategorieName": "Salat",
+                                "kategorieBeschreibung": "Salat"
+                            }
+                        ]
+                    }
+                ]
+                """;
+        //ACT
+        MvcResult result = mockMvc.perform(MockMvcRequestBuilders.get("/api/rezepte"))
+                .andExpect(status().isOk())
+                .andExpect(content().json(expected))
+                .andReturn();
+
+
+        //ASSERT
+        assertEquals(200, result.getResponse().getStatus());
+
+    }
+
+    @Test
+    @DirtiesContext
+    void testCreateRezept() throws Exception {
```

Comment:
Etwas mehr Beschreibung in den Methodennamen wäre ganz nett, damit es etwas einfacher ersichtlich ist, was du hier testen möchtest. Beispiel: createRezeptTest_whenProvideRezept_thenReturnRezeptWithId